src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,12 @@
+.mwFooter {
+  background: #424242;
```

Comment:
I believe there will be a lot of repeated css variables in the future across the app, so I'm suggesting to move things like colors, fonts and font sizes to some variables.scss to share them.

Reply:
Yes I agree, Will do it in next task